pms-115-120/utility/Dockerfile

```diff
@@ -65,10 +65,12 @@ ARG container_user_uid=1002
 ARG container_user_gid=1001
 
 # install packages and create user
-RUN apk -q update \
-&& apk add -q unzip \
-&& addgroup -g ${container_user_gid} ${container_user_group} \
-&& adduser -u /bin/sh -s ${container_user_uid} -G ${container_user_group} -h /home/${container_user} --disabled-password ${container_user}
+RUN apk update \
+&& apk add unzip sudo\
+&& addgroup -g ${container_user_gid} -S ${container_user_group} \
+&& adduser -u ${container_user_uid} -G ${container_user_group} -S ${container_user} -h ${container_user} --disabled-password \
+&& $USER \
+&& sudo echo "%sudo ALL=(ALL) NOPASSWD:/home/${container_user}/${hsm_local_dir}/install.sh" >> /etc/sudoers
```

Comment:
This command will not work for alpine images

```sudo echo "%sudo ALL=(ALL) NOPASSWD:/home/${container_user}/${hsm_local_dir}/install.sh" >> /etc/sudoers```

Reply:
@syedsalman3753 after removing this command docker build is failing. for other alpine image also using same command.